examples/context-propagation/kotlin/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'application'
+    id "org.jetbrains.kotlin.jvm"
+}
+
+dependencies {
+    implementation project(':core')
+
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-jdk8'
+}
+
+application {
+    mainClassName = 'example.armeria.contextpropagation.kotlin.Main'
+}
+
+compileKotlin {
+    kotlinOptions.jvmTarget = "1.8"
```

Comment:
I think this is required in our build because of the way we run tests. Maybe we should apply in top level build file instead?